src/test/resources/alma-fix/990030574430206441.json

```diff
@@ -0,0 +1,121 @@
+    "location" : [ "Napoli" ],
+    "publishedBy" : [ "Libreria scientif. ed" ]
+  } ],
+  "describedBy" : {
+    "id" : "http://lobid.org/resources/990030574430206441",
+    "label" : "Webseite der hbz-Ressource 990030574430206441",
+    "type" : [ "BibliographicDescription" ],
+    "dateCreated" : "20210405",
+    "dateModified" : "20220606",
+    "inDataset" : {
+      "id" : "http://lobid.org/resources/dataset#!",
+      "label" : "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data"
+    },
+    "resultOf" : {
+      "type" : [ "CreateAction" ],
+      "endTime" : "0000-00-00T00:00:00",
+      "instrument" : {
+        "id" : "https://github.com/hbz/lobid-resources",
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "https://lobid.org/hbz01/990030574430206441",
+        "type" : [ "DataFeedItem" ],
+        "label" : "hbz-Ressource 990030574430206441 im Exportformat MARC21 XML",
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        }
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ],
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-HBZ/Off#!",
+      "label" : "lobid Organisation"
+    }
+  },
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT007847893",
+    "label" : "Culturegraph Ressource"
+  }, {
+    "id" : "http://worldcat.org/oclc/1071256649",
+    "label" : "OCLC Ressource"
+  } ],
+  "exampleOfWork" : {
+    "label" : "Carmina",
+    "type" : [ "Work" ]
+  },
+  "extent" : "93 S. ; 8-o",
+  "hasItem" : [ {
+    "id" : "https://lobid.org/item/990030574430206441",
+    "type" : [ "Item", "MBD" ],
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-605#!",
+      "label" : "lobid Organisation"
+    },
+    "label" : "990030574430206441"
+  } ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "type" : [ "BibliographicResource", "Book" ],
+  "responsibilityStatement" : [ "testi per uso accademico. A cura di F[rancesco] Sbordone" ],
+  "contribution" : [ {
+    "agent" : {
+      "gndIdentifier" : "(DE-588)118506080",
+      "id" : "https://d-nb.info/gnd/118506080",
+      "label" : "Bacchylides",
+      "type" : [ "Person" ],
+      "dateOfBirth" : "-0516",
+      "dateOfDeath" : "-0451"
+    },
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/aut",
+      "label" : "Autor/in"
+    },
+    "type" : [ "Contribution" ]
+  }, {
+    "agent" : {
+      "gndIdentifier" : "(DE-588)118594427",
+      "id" : "https://d-nb.info/gnd/118594427",
+      "label" : "Pindarus",
+      "type" : [ "Person" ],
+      "dateOfBirthAndDeath" : "ca. 522 oder 518 v. Chr.-446 v. Chr."
```

Comment:
and it also covers the newly introduced "dateOfBirthAndDeath".

Reply:
Here it is if the filter works with ca. alone.